src/routes/expenses.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Remove 'use strict'; as it's not necessary in ES6 modules.

Reply:
Linter was arguing for that, I did it only because of that